### DISCUSSION TITLE
[Feat] 历史会话列表增加收藏/置顶按钮

### DISCUSSION THREAD

**Discussion by bmwa0813** [coaidev/coai#288] [Discussions | 交流]
1、当前对话页面增加清空对话按钮。这是使用频率远大于“编辑”按钮的必备功能。当前由于没有此功能，用户只能新开会话。虽然效果相同，但会导致对话列表轻松累计上百条本无需保留的无价值历史会话。

2、历史会话列表需要收藏/置顶功能。当前仅有一键清除所有对话，无法高效的筛选与保留有价值的历史会话。

上述两个功能的同时缺失，起到了互相放大的效果，令当前使用体验差强人意。
建议优先优化此功能💕

[a comment by an automated account is not shown]

**Comment by zmh-program** [comment]
duplicate #144

[a comment by an automated account is not shown]

**Comment by Xuanyu0610** [comment]
增加将某个对话所有内容保存为文档的功能

[a comment by an automated account is not shown]

**Comment by zmh-program** [comment]
> 增加将某个对话所有内容保存为文档的功能

not consider

[a comment by an automated account is not shown]
